ble/services/DFUService.h

```diff
@@ -101,7 +103,15 @@ class DFUService {
                 handoverCallback();
             }
 
-            bootloader_start();
+            // Call bootloader_start implicitly trough a event handler call
+            // it is a work around for bootloader_start not being public in sdk 8.1
+            ble_dfu_t p_dfu;
```

Comment:
A handle need to be passed to bootloader_start in sdk 8.0.1 hence ble_dfu_t need to be populated with a connHandle.